crates/driver/openapi.yml

```
@@ -39,27 +39,33 @@ components:
       description: Description of what price to estimate.
       type: object
       properties:
-        sell_token:
+        sellToken:
           $ref: "#/components/schemas/Address"
-        buy_token:
+        buyToken:
           $ref: "#/components/schemas/Address"
         kind:
           description: |
             - `buy`: in_amount is in buy_token, out_amount is in sell_token
             - `sell`: in_amount is in sell_token, out_amount is in buy_token
```

This needs to be updated slightly as well now:
```suggestion
            - `buy`: amount is in buy_token
            - `sell`: amount is in sell_token
```